forum/qa-plugin/tips-widget/tips-replace-phrases.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+function replacePathHtml($str)
+{
+	while (preg_match('/!PATH_HTML\((.*?)\)!/', $str, $matches) === 1)
```

Comment:
Warto zastosować coś takiego jak Yoda conditions. czyli 1 === $zmienna. Jest wiele debat na ten temat, dlatego tylko zauważam, że warto to stosować.